src/components/MessageList/MessageList.js

```diff
@@ -306,34 +173,28 @@ class MessageList extends PureComponent {
    */
   scrollToTarget = (target, containerEl) => {
     // Moved up here for readability:
-    const isElement = target && target.nodeType === 1,
-      isNumber = Object.prototype.toString.call(target) === '[object Number]';
-
-    if (isElement) {
-      containerEl.scrollTop = target.offsetTop;
-    } else if (isNumber) {
-      containerEl.scrollTop = target;
-    } else if (target === 'bottom') {
-      containerEl.scrollTop =
-        containerEl.scrollHeight - containerEl.offsetHeight;
-    } else if (target === 'top') {
-      containerEl.scrollTop = 0;
-    }
+    const isElement = target && target.nodeType === 1;
+    const isNumber =
+      Object.prototype.toString.call(target) === '[object Number]';
+
+    let scrollTop;
+    if (isElement) scrollTop = target.offsetTop;
+    else if (isNumber) scrollTop = target;
+    else if (target === 'top') scrollTop = 0;
+    else if (target === 'bottom')
+      scrollTop = containerEl.scrollHeight - containerEl.offsetHeight;
+
+    if (scrollTop) containerEl.scrollTop = scrollTop; // eslint-disable-line no-param-reassign
```

Comment:
`scrollTop` will be implicitly cast to `false` if it equals 0 (i.e. what you set it to if `target === 'top'`, so this will behave differently from the previous code.

You can resolve this by checking `if (scrollTop !== undefined)`.

Reply:
good catch! pushed a fix